RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -281,63 +522,9 @@ namespace lst {
                                                                 float& betaIn,
                                                                 float& betaInCut,
                                                                 const float ptCut) {
-    bool isPSIn = (modulesInGPU.moduleType[innerInnerLowerModuleIndex] == lst::PS);
```

Comment:
Yes, I've verified that deleting the cuts had no impact on the performance. Because these cuts were applied after the rz helix cuts, they didn't help with timing either. I've updated the comments in accordance with the changes